docs/dictionary/command/play.lcdoc

```diff
@@ -27,18 +27,24 @@ play audioClip "Trust No One" looping
 Example:
 play "/usr/local/clips/music.aiff"
 
+Example:
+# to stop the currently playing audio clip
+play stop
+
 Parameters:
+type (enum):
```

Comment:
It might be better to change this param name to `clipType` (I think it will be ok, but best to avoid possible conflict with the `Type:` element)